test/processors/concerns/http_files_test.rb

```diff
@@ -34,6 +35,15 @@ def logger
     tmp.must_be_instance_of File
   end
 
+  it 'can download a small file' do
+    stub_request(:get, small_uri.to_s).
+      with(headers: { 'Host' => 'test.prx.org:80' } ).
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.